src/lib/transaction_snark/test/util.ml

```diff
@@ -344,6 +344,10 @@ let permissions_from_update (update : Account_update.Update.t) ~auth =
       ( if Zkapp_basic.Set_or_keep.is_keep update.voting_for then
         default.set_voting_for
       else auth )
+  ; set_timing =
+      ( if Zkapp_basic.Set_or_keep.is_keep update.timing then
+        default.set_voting_for
```

Comment:
it would probably be better to use `default.set_timing` instead of `default.set_voting_for`

Reply:
Woops